neo/NeoSystem.cs

```diff
@@ -34,10 +34,11 @@ public class NeoSystem : IDisposable
         public NeoSystem(Store store)
         {
             this.store = store;
+            Plugin.LoadPlugins(this);
```

Comment:
Some plugins may access `Blockchain` or `LocalNode` when loading.

Reply:
But is Blockchain and LocalNode loading synchronous or asynchronous?
If it is synchronous how will that be possible?

Reply:
> Some plugins may access Blockchain or LocalNode when loading.

They can be changed to do whatever loading initialization work they need to do in the new `OnPluginsLoaded()` event.

Reply:
@erikzhang Would you like me to put this hook back: c3cf527
It would allow plugins that need to access `Blockchain` or `LocalNode` during initialization to be able to do so before `OnPluginsLoaded` is called.

Reply:
The current plugins don't need the `OnNeoSystemInitialized` hook, but I can imagine some plugins might want to ensure some other initialization is done before other plugins have `OnPluginsLoaded` called.

Reply:
I don't like `OnNeoSystemInitialized()`.

Reply:
What do you suggest to allow the genesis block to be persisted after the plugins are loaded? I guess I could Ask `Blockchain` to persist the genesis block after the plugins are loaded, but then some plugins on initialization might need the genesis block to already be there. This solution solves that problem.